### ISSUE TITLE
Publish schemas to a repository

### ISSUE THREAD

**Issue by cmarchand** [xproc/3.0-grammar#3]
If all schemas were published to a central repository, maven central for example, it'll be a help for people who try to develop editors, or for implementors.

**Comment by ndw** [comment]
Can you usefully publish just data to Maven? Can you give me an example of a package that does?

**Comment by cmarchand** [comment]
No. Well, at least, you publish artifacts ; and artifacts may be other file types than jar files. For examples, most of open-source projects publish sources and javadoc to maven. It's packaged as jar, but does not contains any .class files.

I'm not sure the way you want to do it, but when I have to publish various things to maven, I use [build-helper:attach-artifact](https://www.mojohaus.org/build-helper-maven-plugin/attach-artifact-mojo.html) to attach additional artifacts to publish. 

If you look at [maven artifact definition](https://maven.apache.org/ref/3.5.4/maven-artifact/apidocs/org/apache/maven/artifact/DefaultArtifact.html#DefaultArtifact-java.lang.String-java.lang.String-java.lang.String-java.lang.String-java.lang.String-java.lang.String-org.apache.maven.artifact.handler.ArtifactHandler-) you see standard groupId, artifactId, version, scope to define the use, and more important, **type** and **classifier**.

- `type` is the file type : jar / zip / tar.gz, and so on... It isn't [well documented](https://maven.apache.org/ref/3.5.4/maven-artifact/apidocs/org/apache/maven/artifact/Artifact.html#getType--) in sources, but [better in documentation reference](http://maven.apache.org/pom.html#Dependencies)
- `classifier` is the role : sources / javadoc, and so on.

You may add as many artifact as you want, and (artifacts, pom, checksum) is the component.

You may also have a look at [deploy:deploy-file](https://maven.apache.org/plugins/maven-deploy-plugin/deploy-file-mojo.html) who allows you to deploy one or many files to a remote repository, outside of a maven project. Better, it's able to generate the pom.xml file if you do not provide one. It may be useful in this case.

**Comment by cmarchand** [comment]
Re-reading your question and my response, deploy:deploy-jar should do the trick...

**Comment by ndw** [comment]
Moved to https://codeberg.org/XProc/specification/issues/96
